tests/Google/AccessToken/RevokeTest.php

```diff
@@ -23,128 +23,133 @@
-      ->method('createRequest')
-      ->will($this->returnCallback(
-            function ($method, $url, $params) use (&$requestToken, $request) {
-              parse_str((string) $params['body'], $fields);
-              $requestToken = isset($fields['token']) ? $fields['token'] : null;
-
-              return $request;
-            }
-        ));
-
-    $t = array(
-      'access_token' => $accessToken,
-      'created' => time(),
-      'expires_in' => '3600'
-    );
-
-    // Test with access token.
-    $revoke = new Google_AccessToken_Revoke($http);
-    $this->assertTrue($revoke->revokeToken($t));
-    $this->assertEquals($accessToken, $token);
-
-    // Test with refresh token.
-    $revoke = new Google_AccessToken_Revoke($http);
-    $t = array(
-      'access_token' => $accessToken,
-      'refresh_token' => $refreshToken,
-      'created' => time(),
-      'expires_in' => '3600'
-    );
-    $this->assertTrue($revoke->revokeToken($t));
-    $this->assertEquals($refreshToken, $token);
-
-    // Test with token string.
-    $revoke = new Google_AccessToken_Revoke($http);
-    $t = $accessToken;
-    $this->assertTrue($revoke->revokeToken($t));
-    $this->assertEquals($accessToken, $token);
-  }
-
-  public function testRevokeAccessGuzzle6()
-  {
-    $this->onlyGuzzle6();
-
-    $accessToken = 'ACCESS_TOKEN';
-    $refreshToken = 'REFRESH_TOKEN';
-    $token = '';
-
-    $response = $this->getMock('Psr\Http\Message\ResponseInterface');
-    $response->expects($this->exactly(3))
-      ->method('getStatusCode')
-      ->will($this->returnValue(200));
-    $http = $this->getMock('GuzzleHttp\ClientInterface');
-    $http->expects($this->exactly(3))
-      ->method('send')
-      ->will($this->returnCallback(
-            function ($request) use (&$token, $response) {
-              parse_str((string) $request->getBody(), $fields);
-              $token = isset($fields['token']) ? $fields['token'] : null;
-
-              return $response;
-            }
-        ));
-
-    $t = array(
-      'access_token' => $accessToken,
-      'created' => time(),
-      'expires_in' => '3600'
-    );
-
-    // Test with access token.
-    $revoke = new Google_AccessToken_Revoke($http);
-    $this->assertTrue($revoke->revokeToken($t));
-    $this->assertEquals($accessToken, $token);
-
-    // Test with refresh token.
-    $revoke = new Google_AccessToken_Revoke($http);
-    $t = array(
-      'access_token' => $accessToken,
-      'refresh_token' => $refreshToken,
-      'created' => time(),
-      'expires_in' => '3600'
-    );
-    $this->assertTrue($revoke->revokeToken($t));
-    $this->assertEquals($refreshToken, $token);
-
-    // Test with token string.
-    $revoke = new Google_AccessToken_Revoke($http);
-    $t = $accessToken;
-    $this->assertTrue($revoke->revokeToken($t));
-    $this->assertEquals($accessToken, $token);
-  }
+    public function testRevokeAccessGuzzle5()
+    {
+        $this->onlyGuzzle5();
```

Comment:
Why did you fix the spaces here but not in the other tests? Is it because you realized that arduous task would crush your soul?

FWIW I think this is great, and if we want we can have a follow-up PR for _just_ converting 2 spaces to 4 spaces

Reply:
yes, that's a fairly good description of what happened.